### DISCUSSION TITLE
How do you convert sliders to streams?

### DISCUSSION THREAD

**Discussion by localcake** [ppy/osu#14245] [Q&A] [answered]
In stable the hotkey was Ctrl+Shift+F but now that does nothing.

**Comment by smoogipoo** [comment, marked as answer]
This doesn't exist yet.

**Reply by localcake** [reply]
Does testing in the editor also not exist yet?

**Reply by smoogipoo** [reply]
That also doesn't exist https://github.com/ppy/osu/issues/7008

**Reply by frenzibyte** [reply]
Issue opened at https://github.com/ppy/osu/issues/14246
